api/v1/client/cilium_client.go

```diff
@@ -0,0 +1,78 @@
+package client
```

Comment:
I think this is a non trivial matter. I want to understand this better before we blindly attach a license to generated code.